src/sagemaker/utils.py

```diff
@@ -344,6 +344,17 @@ def repack_model(inference_script, source_directory, model_uri, sagemaker_sessio
 
         shutil.copytree(dirname, code_dir)
 
+        default_inference_script = '''
+from __future__ import absolute_import
```

Comment:
format?

Reply:
I think it might be better to make this its own file and move it?

Reply:
I personally prefer writing these 5 lines of code that will write 2 lines of code into a file than creating a file template that needs to copied into the library and write code to use populate that template with the module name.

Reply:
What do you mean by **format?**

Reply:
look at the code file, you have code that is indented properly, and then this line is just there. lol